submissions/xT0PERx/js-dom/visuality.css

```diff
@@ -0,0 +1,79 @@
+* {
```

Comment:
why do you need it?

Reply:
This way I want to make the padding the same for all my elements. Perhaps using "body" instead of "*" would have been more correct, but perhaps due to my inexperience I decided to use "*".

Reply:
U can read this articles 
https://htmlacademy.ru/blog/boost/frontend/short-13
https://htmlacademy.ru/blog/boost/frontend/about-normalize-css

its can help you chose better way to normalize your page.

Reply:
This way I want to make the padding the same for all my elements. Perhaps using "body" would have been more correct.